index.html

```diff
@@ -0,0 +1,196 @@
+                <img src="img/Avatar.png" alt="avatar">
+            </button>
+        </div>
+    </header>
+    <main>
+        <h1>Contact us</h1>
+        <p>Need an experienced and skilled hand with custom IT projects?</p>
+        <p>Fill out the form to get a free consultation.</p>
+        <form action="#" method="post">
+            <div>
+                <div>
+                    <div>
+                        <label for="company">Your company name*</label>
+                        <input type="text" name="company" id="company" required>
+                    </div>
+                    <div>
+                        <label for="business">Nature of Business*</label>
+                        <input type="text" name="business" id="business" required>
+                    </div>
+                    <div>
+                        <label for="data">Data of registration</label>
+                        <input type="date" placeholder="dd/mm/yyyy" name="data" id="data">
+                    </div>
+                </div>
+                <div>
+                    <div>
+                        <label for="country">Country of registration*</label>
+                        <select name="country" id="country" required>
+                            <option value="1">Select</option>
+                            <option value="2">Option 01</option>
+                            <option value="3">Option 02</option>
+                            <option value="4">Option 03</option>
+                        </select>
+                    </div>
+                    <div>
+                        <label for="address">Address</label>
+                        <input type="text" name="address" id="address">
+                    </div>
+                    <div>
+                        <label for="postcode">Postcode</label>
+                        <input type="text" name="postcode" id="postcode">
+                    </div>
+                </div>
+            </div>
+            <hr>
+            <div>
+                <div>
+                    <div>
+                        <label for="name">Contact name</label>
+                        <input type="text" name="name" id="name">
+                    </div>
+                    <div>
+                        <label for="phone">Contact phone</label>
+                        <input type="tel" name="phone" id="phone">
+                    </div>
+                    <div>
+                        <label for="email">Email*</label>
+                        <input type="email" name="email" id="email" required>
+                    </div>
+                </div>
+                <div>
+                    <label for="idea">Let’s talk about your idea</label>
+                    <textarea name="idea" id="idea" cols="30" rows="10" maxlength="320"></textarea>
+                    <div>
+                        <div>
+                            <input type="range" min="0" max="100" step="100" list="tickmarks">

```

Comment:
```suggestion
                            <datalist > 
                                <option value="0" label="0">
                                <option value="10">
                                <option value="20">
                                <option value="30">
                                <option value="40">
                                <option value="50">
                                <option value="60">
                                <option value="70">
                                <option value="80">
                                <option value="90">
                                <option value="100" label="100">
                            </datalist>

```

Please don't mix up TAG `<label>` where we need an attribute 'for' and an ATTRIBUTE 'label' which is used for option in your case